velox/exec/tests/AssignUniqueIdTest.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/exec/tests/OperatorTestBase.h"
+#include "velox/exec/tests/PlanBuilder.h"
+#include "velox/exec/tests/QueryAssertions.h"
+#include "velox/functions/prestosql/tests/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec::test;
+using functions::test::FunctionBaseTest;
+
+class AssignUniqueIdTest : public OperatorTestBase {
+ protected:
+  void assertUniqueIdAdded(
+      const std::shared_ptr<const core::PlanNode>& plan,
+      const std::vector<RowVectorPtr> inputRows) {
+    CursorParameters params;
+    params.planNode = plan;
+    auto result = readCursor(params, [](auto /*task*/) {});
+    auto numColumns = result.second[0]->childrenSize();
+    ASSERT_EQ(numColumns, inputRows[0]->childrenSize() + 1);
+
+    for (int i = 0; i < numColumns; i++) {
+      auto column = result.second[0]->childAt(i);
+      if (i < numColumns - 1) {
+        FunctionBaseTest::assertEqualVectors(inputRows[0]->childAt(i), column);
```

Comment:
FunctionBaseTest class has a specific purpose of being a base class for tests for various functions. This is rather short method. Can we just copy-paste it into OperatorTestBase?